src/plugins/trivia.js

```diff
@@ -132,6 +192,30 @@ async function answer(message, params) {
   }
 }
 
+// player votes to increase trivia hints
+async function hint(message) {
+  await db.voteIncreaseTriviaHintLevel(message.author.id);
+
+  const voters = await db.getTriviaHintVoteCount();
+
+  // minimum of 2 players agree to increase hint level
+  if (voters >= 2) {
+    await db.resetTriviaHintVoteCount();
+    await db.resetTriviaSkip();
+    await db.increaseTriviaHintLevel();
```

Comment:
Same three calls in a row again, extract?

Reply:
i agree with the rest, but not this one because it needs to reset vote and skip counting, but not hint level